spec/system/candidate_interface/candidate_account_spec.rb

```diff
@@ -40,6 +40,14 @@
 
     when_i_signed_in_more_than_a_week_ago
     then_i_should_be_signed_out
+
+    when_i_click_the_signin_link
+    and_i_submit_my_email_address(uppercase: true)
```

Comment:
These steps should read as English if possible, perhaps this can be:

```suggestion
    and_i_submit_my_email_address_in_uppercase
```